src/Mod/PartDesign/App/FeatureLinearPattern.cpp

```diff
@@ -72,12 +72,16 @@ short LinearPattern::mustExecute() const
 
 const std::list<gp_Trsf> LinearPattern::getTransformations(const std::vector<App::DocumentObject*>)
 {
-    double distance = Length.getValue();
-    if (distance < Precision::Confusion())
-        throw Base::ValueError("Pattern length too small");
     int occurrences = Occurrences.getValue();
+    if (occurrences <= 1)
+
+        return {gp_Trsf()};
     if (occurrences < 1)
         throw Base::ValueError("At least one occurrence required");

```

Comment:
```suggestion
    if (occurrences < 1)
        throw Base::ValueError("At least one occurrence required");

    if (occurrences == 1)
        return {gp_Trsf()};
```
Sorry @Jolbas I was too fast at reviewing and mixed with some other stuffs. :confused: 
I think it's better as suggested above. Sorry again.

Reply:
Yes. No problem. I should have reviewed your suggestion but did just accept it.